lib/ansible/modules/windows/win_dns_record.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Hitachi ID Systems, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a windows documentation stub. The actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_dns_record
+version_added: "2.8"
+short_description: Manage Windows Server DNS records
+description:
+- Manages DNS records within an existing Windows Server DNS zone
```

Comment:
All descriptions, except the short_description, must end with a trailing dot.
```suggestion
- Manages DNS records within an existing Windows Server DNS zone.
```